chapter_1/Exeption handling/3°_tryal.py

```diff
@@ -0,0 +1,4 @@
+try:
+    print("{+} 1337/0 = " + str(1337/0))
+except Exception as e:
+    print("{-} Error = "+str(e))
```

Comment:
Please add spaces next to the `+`, to follow PEP8 style guide.

```suggestion
    print("{-} Error = " + str(e))
```

Reply:
Done.